cmd.go

```diff
@@ -223,7 +223,12 @@ func doBuild(args []string) {
 		goArgs = mapToTmpDir(tmpDir, goArgs)
 	}
 
-	bin := filepath.Base(strings.TrimSuffix(goArgs[0], ".go")) + ".debug"
+	exeSuffix := ""
```

Comment:
What about `binSuffix` so that we can leave the code open to other executable binary file extensions that `go` may produce in the future?